control-plane-operator/controllers/hostedcontrolplane/v2/kas/component.go

```diff
@@ -101,14 +101,16 @@ func NewComponent() component.ControlPlaneComponent {
 			"aws-pod-identity-webhook-kubeconfig.yaml",
 			component.EnableForPlatform(hyperv1.AWSPlatform),
 			component.WithAdaptFunction(adaptAWSPodIdentityWebhookKubeconfigSecret),
+			component.ReconcileExisting(),
 		).
 		WithManifestAdapter(
 			"azure-kms-secretprovider.yaml",
 			component.WithAdaptFunction(kms.AdaptAzureSecretProvider),
 			component.WithPredicate(enableAzureKMSSecretProvider),
 		).
-		RolloutOnConfigMapChange("kas-config", "kas-audit-config", "auth-config").
-		RolloutOnSecretChange("kas-secret-encryption-config").
+		ExcludeConfigMapsFromConfigHash(
+			"client-ca",
```

Comment:
I don't think the individual components should decide whether to exclude this configMap or not, I think we should just always exclude it as it will cause any component to keep restarting.  wdyt?

Reply:
sgtm, will update